src/Kunstmaan/AdminBundle/Form/MediaTokenTransformer.php

```diff
@@ -0,0 +1,72 @@
+<?php
+
+namespace Kunstmaan\AdminBundle\Form;
+
+use Symfony\Component\DomCrawler\Crawler;
+use Symfony\Component\Form\DataTransformerInterface;
+
+/**
+ * Class MediaTokenTransformer.
+ */
+class MediaTokenTransformer implements DataTransformerInterface
+{
+    /**
+     * @param mixed $content
+     *
+     * @return string
+     */
+    public function transform($content)
+    {
+        $html = utf8_encode("<!DOCTYPE html>
```

Comment:
does this really work? the native utf8 support for PHP is rather shady. 

When using the crawler, I always use `addHTMLContent`, and provide the charset myself, or add a `<meta charset=utf8>` to the HTML string as in your case.